### ISSUE TITLE
Why was sponsor info removed?

### ISSUE THREAD

**Issue by BE-CH** [ShareDropio/sharedrop#179]
Hi

I would like to sponsor this project using Github Sponsors but i see its not possible. Any plans for this in the future?

Furthermore, why was sponsor info removed in commit 85223ea?

**Comment by lukebouch** [comment]
I think #182 has something to do with this.

**Comment by BE-CH** [comment]
@lukebouch makes sense! I am swapping to https://github.com/schlagmichdoch/pairdrop.

Thanks!
